app/controllers/integrations/amazon_basecamp_controller.rb

```diff
@@ -25,10 +21,10 @@ def create
     end
 
     if amz_message_type.to_s.downcase == 'notification'
-      # DO WORK HERE
-      message = "Subject: #{request_body['Subject']}<br/> #{request_body['Message']}"
-      # send message to basecamp
-      HTTParty.post basecampbot_url, query: { content: message }
+      ChatbotService.new(Rails.configuration.service['basecampbot_url']).post_message(
```

Comment:
1. Why create an instnace with `new` when it is more or less stateless?
2. Why have `service` in the name of the class?